metricflow/sql/sql_exprs.py

```diff
@@ -145,6 +145,10 @@ def contains_column_alias_exprs(self) -> bool:  # noqa: D
     def contains_ambiguous_exprs(self) -> bool:  # noqa: D
         return self.contains_string_exprs or self.contains_column_alias_exprs
 
+    @property
+    def contains_aggregate_exprs(self) -> bool:  # noqa: D
+        return any(isinstance(x, SqlAggregateFunctionExpression) for x in self.function_exprs)
```

Comment:
Would like to avoid the use of `isinstance`. Maybe add a boolean property like `is_aggregate_function` or similar to `SqlFunctionExpression`.

Reply:
fixed